app/Http/Controllers/Tabbar/TabbarController.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Http\Controllers\Tabbar;
+
+use App\Http\Controllers\Controller;
+use App\Settings\PathInfo;
+
+class TabbarController extends Controller
+{
+    /*
+    |--------------------------------------------------------------------------
+    | TabbarController
+    |--------------------------------------------------------------------------
+    |
+    | Tabbarコントローラ
+    |
+    */
+
+    /**
+     * Create a new tabbar controller instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        // $pathInfo = new PathInfo();
+        // $pathInfo->getNodePath();
+
+    }
+
+    /**
+     * topアクション
+     *
+     * @return view tabbar/tabbar_topテンプレート
+     */
+    public function topAction()
+    {
+    	return view('tabbar/tabbar_top');
+    }
+
+}
```

Comment:
__[code_sniffer]__
* The closing brace for the class must go on the next line after the body